File: daySix/Javascript/QuestionFour.js
let sortArrayOf=(arrayOfNumbers)=>{
    let store = 0;
    
    for(let count = 0; count < arrayOfNumbers.length; count++){
        for(let counter = 0; counter < arrayOfNumbers.length; counter++){
            if(arrayOfNumbers[count] > arrayOfNumbers[counter]){
                store = arrayOfNumbers[counter];
                arrayOfNumbers[counter] = arrayOfNumbers[count];
                arrayOfNumbers[count] = store;
            }
        }
            
    }
    
return arrayOfNumbers;
}


let countDuplicateElementsIn=(arrayOfNumbers)=>{
   
   let duplicateCounter = 0;
   let counter = 0;
   let store = 0;
   let sortArray = sortArrayOf(arrayOfNumbers);
   
   for(let index = 0; index < sortArray.length; index++){
           counter = 0;
       for(let count = 0; count < sortArray.length; count++){
           if(sortArray[index] == sortArray[count] && sortArray[index] != store){
               counter++;
           }
       }
       if(counter > 1){
           duplicateCounter++;
           store = sortArray[index];
       }
           
   }
   
return duplicateCounter;
}

let getArrayOfDuplicateElement=(arrayOfNumbers)=>{
   
   let counter = 0;
   let store = 0;
   let sortArray = sortArrayOf(arrayOfNumbers);
   let duplicateElementArray = [];
   
   for(let index = 0; index < sortArray.length; index++){
           counter = 0;
       for(let count = 0; count < sortArray.length; count++){
           if(sortArray[index] == sortArray[count] && sortArray[index] != store){
               counter++;
           }
       }
       if(counter > 1){
           store = sortArray[index];
           duplicateElementArray.push(sortArray[index]);
       }
           
   }
   
return duplicateElementArray;
}


arrayOfNumbers = [45,60,3,0,67,2,45,3,22,0];

console.log(getArrayOfDuplicateElement(arrayOfNumbers));

File: daySix/python/question_four.py
def sort_array_of(array_of_numbers):
    store = 0
    
    for count in range(len(array_of_numbers)):
        for counter in range(len(array_of_numbers)):
            if(array_of_numbers[count] > array_of_numbers[counter]):
                store = array_of_numbers[counter]
                array_of_numbers[counter] = array_of_numbers[count]
                array_of_numbers[count] = store
    
    return array_of_numbers



def count_duplicate_elements_in(array_of_numbers):
   
   duplicate_counter = 0
   counter = 0
   store = 0
   sort_array = sort_array_of(array_of_numbers)
   
   for index in range(len(sort_array)):
        counter = 0
        for count in range(len(sort_array)):
            if(sort_array[index] == sort_array[count] and sort_array[index] != store):
               counter += 1
           
        if(counter > 1):
           duplicate_counter += 1
           store = sort_array[index]
   return duplicate_counter


def get_array_of_duplicate_element(array_of_numbers):
   
   counter = 0
   store = 0
   sort_array = sort_array_of(array_of_numbers)
   duplicate_element_array = []
   
   for index in range(len(sort_array)):
        counter = 0
        for count in range (len(sort_array)):
           if(sort_array[index] == sort_array[count] and sort_array[index] != store):
               counter += 1
         
        if(counter > 1):
           store = sort_array[index]
           duplicate_element_array.append(sort_array[index])
           index_counter += 1
        
        
        return duplicate_element_array;

